gateway/modules/auth/match_types.go

```diff
@@ -13,32 +13,32 @@ func matchString(rule *config.Rule, args map[string]interface{}) error {
 	var f2 string
 	f1String, ok := rule.F1.(string)
 	if !ok {
-		return ErrIncorrectRuleFieldType
+		return formaterror(rule, ErrIncorrectRuleFieldType)
```

Comment:
Remove all format error statements from this file. It will get handled in the `match` rule handler itself